packages/storybook/layouts.stories.js

```diff
@@ -113,54 +113,88 @@ storiesOf('Layouts', module)
       </cxl-layout-marketing>
     `;
   })
-  .add('cxl-layout-institute', () => {
+  .add('cxl-institute-layout', () => {
     return html`
-      <cxl-layout-institute>
-        <div class="wrapper">
-          <div id="left-nav">
-            <a href="/">
-              <div class="logo">
-                <span class="cxl">C<span>XL</span></span>
-              </div>
-            </a>
-            <div class="top-nav">
-              <ul class="nav-items">
-                <li>
-                  <a href="trainings" class="active">
-                    <iron-icon class="size-l" icon="cxl:nav-trainings"></iron-icon>
-                    <iron-icon class="size-l" icon="cxl:nav-trainings-active"></iron-icon>
-                    <span>Trainings</span>
-                  </a>
-                </li>
-                <li>
-                  <a href="roadmap">
-                    <iron-icon class="size-l" icon="cxl:nav-roadmap"></iron-icon>
-                    <iron-icon class="size-l" icon="cxl:nav-roadmap-active"></iron-icon>
-                    <span>Discover</span>
-                  </a>
-                </li>
-                <li>
-                  <a href="discover">
-                    <iron-icon class="size-l" icon="cxl:nav-discover"></iron-icon>
-                    <iron-icon class="size-l" icon="cxl:nav-discover-active"></iron-icon>
-                    <span>Manage</span>
-                  </a>
-                </li>
-              </ul>
-            </div>
-            <div class="bottom-nav">
-              <ul class="nav-items">
-                <li>
-                  <a href="/">
-                    <iron-icon class="size-l" icon="lumo:user"></iron-icon>
-                    <span>Profile</span>
-                  </a>
-                </li>
-              </ul>
+      <div class="wrapper">
```

Comment:
It's always `.wrap`, no extra innovation needed.

Reply:
.wrap is container that sets padding from both side and positions content to center with fixed width. If you look closer it's wrapper for left-nav and cxl-institute-layout.